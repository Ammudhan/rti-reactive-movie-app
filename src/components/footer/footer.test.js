import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Footer from './footer';

describe('Footer Component', () => {

    it('should renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Footer />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('should matches snapshot', () => {
        const
            tree = renderer.create(
                <Footer />
            ).toJSON();
        expect(tree).toMatchSnapshot();
    });

});