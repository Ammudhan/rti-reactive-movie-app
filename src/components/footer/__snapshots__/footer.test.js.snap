// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should matches snapshot 1`] = `
<span
  className="row"
>
  <span
    className="col-12 page-layout d-flex align-items-center justify-content-center"
  >
    <h1
      className="footer-title"
    >
      Footer
    </h1>
  </span>
</span>
`;

exports[`Footer matches snapshot 1`] = `
<span
  className="row"
>
  <span
    className="col-12 page-layout d-flex align-items-center justify-content-center"
  >
    <h1
      className="footer-title"
    >
      Footer
    </h1>
  </span>
</span>
`;
